src/components/Screen/index.js

```diff
@@ -17,13 +17,16 @@ const Split = ({
 
     <div className='flex-ns'>
 
-      <main className='mr4-ns w-50-ns'>
+      <main className={`mr4-ns ${aside ? 'w-70-ns' : 'w-100-ns'}`}>
         {main}
       </main>
 
-      <aside className='w-50-ns mt5 mt0-ns'>
-        {aside}
-      </aside>
+      {aside
+        ? <aside className='w-30-ns mt5 mt0-ns'>
+            {aside}
+          </aside>
+        : null
+      }
```

Comment:
aside is now optional, so a `Screen` is full-width on desktop if `aside` is not given; everything is the same on mobile (vertically stacked).
![image](https://cloud.githubusercontent.com/assets/5497885/23436714/d0ae0ba8-fdc8-11e6-831c-3b2ac472bc19.png)